Examples-Web/IdleTimeout/package.json

```diff
@@ -0,0 +1,66 @@
+{
+    "name": "touch_free_react",
```

Comment:
Might be worth updating this so we don't end up with tons of packages all called the same thing!